tools/mksurfdata_esmf/src/mksoiltexMod.F90

```diff
@@ -514,10 +428,11 @@ subroutine mksoiltex(file_mesh_i, file_mapunit_i, file_lookup_i, mesh_o, pioid_o
           ! (calculated from orgc_i, cfrag_i, and bulk_i) to organic_o. This
           ! approach first calculates organic_i and then regrids to organic_o
           ! rather than regridding all the terms first and then calculating
-          ! organic_o. Commented out code above belongs to that option.
+          ! organic_o. That would be organic_o_option 2, last available in
+          ! commit d5f389a97.
```

Comment:
```suggestion
          ! organic_o. This option was last available in commit d5f389a97.
```

Reply:
I like the original text more, because it tells the reader what to look for when they check out that commit.